tutorials/abap-environment-business-service-provisioning/abap-environment-business-service-provisioning.md

```diff
@@ -1,7 +1,7 @@
 ---
 auto_validation: true
 title: Expose a Standard Core Data Service for ABAP Environment
-description: Create a service definition, service binding and communication scenario to expose a Core Data Services (CDS) view in SAP Cloud Platform ABAP Environment.
+description: Create a service definition, service binding and communication scenario to expose a Core Data Services (CDS) view in SAP BTP ABAP Environment.
```

Comment:
Please use the full name "SAP Business Technology Platform" at least once at the beginning of your tutorial before you use the abbreviation.